x-pack/plugins/security_solution/public/timelines/components/timeline/unified_components/index.test.tsx

```diff
@@ -0,0 +1,629 @@
+            sortDirection: 'asc',
+            columnId: 'event.severity',
+            columnType: 'number',
+          },
+        ]);
+      });
+    });
+
+    it('should be able to edit DataView Field', async () => {
+      const field = {
+        name: 'message',
+      };
+      const { container } = renderTestComponents();
+
+      await waitFor(() => {
+        expect(screen.getByTestId('discoverDocTable')).toBeVisible();
+      });
+      expect(container.querySelector(`[data-gridcell-column-id="${field.name}"]`)).toHaveAttribute(
+        'data-gridcell-column-index',
+        '3'
+      );
+
+      expect(
+        container.querySelector(`[data-gridcell-column-id="${field.name}"]`)
+      ).toBeInTheDocument();
+
+      fireEvent.click(
+        container.querySelector(
+          `[data-gridcell-column-id="${field.name}"] .euiDataGridHeaderCell__icon`
+        ) as HTMLElement
+      );
+
+      await waitFor(() => {
+        expect(screen.getByTitle('Edit data view field')).toBeEnabled();
+      });
+
+      fireEvent.click(screen.getByTitle('Edit data view field'));
+
+      await waitFor(() => {
+        expect(kibanaServiceMock.dataViewFieldEditor.openEditor).toHaveBeenNthCalledWith(1, {
+          ctx: {
+            dataView: expect.objectContaining({
+              id: 'security-solution',
+            }),
+          },
+          fieldName: 'message',
+          onSave: expect.any(Function),
+        });
+      });
+    });
+  });
+
+  describe('unified field list', () => {
+    it('should be able to add filters', async () => {
+      const field = {
+        name: 'event.severity',
+      };
+
+      renderTestComponents();
+      expect(await screen.findByTestId('timeline-sidebar')).toBeVisible();
+
+      await waitFor(() => {
+        expect(screen.getByTestId('fieldListGroupedAvailableFields-count')).toBeVisible();
+      });
+
+      expect(screen.queryAllByTestId(`dataGridHeaderCell-${field.name}`)).toHaveLength(1);
+
+      const availableFields = screen.getByTestId('fieldListGroupedAvailableFields');
+
+      fireEvent.click(within(availableFields).getByTestId(`field-${field.name}-showDetails`));
+
+      await waitFor(() => {
+        expect(screen.getByTestId(`timelineFieldListPanelAddExistFilter-${field.name}`));
+      });
+
+      fireEvent.click(screen.getByTestId(`timelineFieldListPanelAddExistFilter-${field.name}`));
+      await waitFor(() => {
+        expect(mockTimelineFilterManager.addFilters).toHaveBeenNthCalledWith(
+          1,
+          expect.arrayContaining([
+            expect.objectContaining({
+              query: {
+                exists: {
+                  field: field.name,
+                },
+              },
+              meta: expect.objectContaining({
+                negate: false,
+                disabled: false,
+              }),
+            }),
+          ])
+        );
+      });
+    });
+    it('should remove the column when clicked on ⊕ sign', async () => {
```

Comment:
what sign is this?

Reply:
Also, isn't this testing adding the column?

Reply:
yes.. corrected the wording. Below is the same sign in unified field list which is being tested. But for simplicity I have changed ⨁ --> + in the test description.

![grafik](https://github.com/elastic/kibana/assets/7485038/86b3dc07-385b-4fff-8157-eba2033442d1)

Reply:
Ahh...okay, that makes sense. Thanks!